cfme/utils/net.py

```diff
@@ -165,17 +165,22 @@ def find_pingable(mgmt_vm):
          In priority: first pingable address, address 'selected' by wrapanapi (possibly None)
      """
     for ip in getattr(mgmt_vm, 'all_ips', []):
-        if not ip.lower().startswith('fe80::') and is_pingable(ip):
-            logger.info('Found reachable IP for VM: %s', ip)
-            return ip
-        else:
+        if ip.lower().startswith('fe80::') or (not allow_ipv6 and is_ipv6(ip)):
```

Comment:
Should `ip.lower().startswith('fe80::')` be part of is_ipv6? Is there a reason it isn't?